stylesheets/_component.piano.scss

```diff
@@ -1,27 +1,26 @@
+$piano-bg: #fafafa;
```

Comment:
We do have `color(background, light)` which is 2% lighter than off-white, if any good?

Reply:
Thanks, I tried that but it was tool close to the `.ui-report` background colour.